packages/oi4-oec-service-opcua-model/src/opcua/OPCUABuilder.ts

```diff
@@ -147,11 +150,7 @@ export class OPCUABuilder {
       correlationId: correlationId,
       MetaData: opcUaMetaDataPayload,
     };
-    if (this.lastMessageId === opcUaMetaDataMessage.MessageId) {
-      opcUaMetaDataMessage.MessageId = `OverFlow${opcUaMetaDataMessage.MessageId}`;
-    } else {
-      this.lastMessageId = opcUaMetaDataMessage.MessageId;
-    }
+    this.setAtomarLastMessageId(opcUaMetaDataMessage);
```

Comment:
use something like 
```suggestion
    const messageId =  getUniqueMessageId(`${Date.now().toString()}-${this.publisherId}`)
```
use it in before the initialization of the  IOPCUAMetaData in line 143 and pass the value directly. This avoid the changing of the value
Furthermore